src/test/suite/didact.test.ts

```diff
@@ -264,6 +268,27 @@ suite('Didact test suite', () => {
 			}			
 		});
 	});
+
+	test('make sure we can open a didact file from a vscode extension', async () => {
+		await extensionFunctions.handleVSCodeUri(testMD5);
+		await delay(1000);
+		if (didactManager.active()) {
+			const title : string | undefined = didactManager.active()?.getCurrentTitle();
+			expect(title).to.equal('Didact Terminal Commands');
+		}
```

Comment:
in cas the didactManager is no tactive the test will pass withtou checking that it is really working